External/FEXCore/Source/Interface/Context/Context.h

```diff
@@ -104,6 +104,7 @@ namespace FEXCore::Context {
       FEX_CONFIG_OPT(BlockJITNaming, BLOCKJITNAMING);
       FEX_CONFIG_OPT(ParanoidTSO, PARANOIDTSO);
       FEX_CONFIG_OPT(CacheObjectCodeCompilation, CACHEOBJECTCODECOMPILATION);
+      FEX_CONFIG_OPT(x87ReducedPrecision, X87REDUCEDPRECISION);
```

Comment:
This seems unrelated?

Reply:
ah used for context format. We should also add to AOTIR.

Reply:
It is necessary to have for `CodeObjectSerializationConfig` which pulls all of its configuration features from the Context Config location. Anything that effects codegen needs to be generated in that hash object.